deployment/dockerfiles/origin-dapp2

```diff
@@ -32,11 +30,24 @@ COPY ./origin-contracts/releases/0.8.5/build/ ./origin-contracts/build/
 
 RUN npm run build --prefix experimental/origin-dapp2
 
+# Conditional copy of config depending on NAMESPACE build arg using multi stage
+# builds
+FROM build as build-dev
 RUN mv /app/experimental/origin-dapp2/public/kovan.html \
```

Comment:
Where are those kovan|rinkeby|mainnet.html files coming from ?
I don't see them under https://github.com/OriginProtocol/origin/tree/master/experimental/origin-dapp2/public ?

Reply:
oh I see. they are generated by webpack. But then where does it get the blockchain network id ? Like 4 for rinkeby ? Sorry I' haven't looked at dapp2 much yet...

Reply:
It loads the configuration from origin-graphic/src/contracts.js. That file has hardcoded contract addresses so it doesn’t need a network ID like we had when the contract addresses were read out of the contract build files.